service/create/third_party_resource.go

```diff
@@ -0,0 +1,27 @@
+package create
+
+import (
+	"github.com/giantswarm/certificatetpr"
+	microerror "github.com/giantswarm/microkit/error"
+	"k8s.io/client-go/pkg/api/errors"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/apis/extensions/v1beta1"
+)
+
+func (s *Service) createTPR() error {
+	tpr := &v1beta1.ThirdPartyResource{
+		ObjectMeta: v1.ObjectMeta{
+			Name: certificatetpr.Name,
+		},
+		Versions: []v1beta1.APIVersion{
+			{Name: "v1"},
+		},
+		Description: "Managed certificates on Kubernetes clusters",
+	}
+	_, err := s.Config.K8sClient.Extensions().ThirdPartyResources().Create(tpr)
+	if err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
I would find it cool if the conditionals would be more declarative and separated like this. It is easier for my weak brain to read it. 

```
if errors.AlreadyExists(err) {
    return nil
} else if err != nil {
    return microerror.MaskAny(err)
}
```